INSTALL.md

```diff
@@ -8,6 +8,10 @@ INSTALL/UPGRADE VM
     
     > vagrant plugin install vagrant-vbguest
     
+    Update your dotfiles in your own `~` directory 
+    
+    Update the `config.dist.yaml` file as needed (primarily memory and shared files location)  There is no longer a merge between config files.
```

Comment:
Yeah, I fixed it pretty much as we were talking about it.

On Tue, Jun 9, 2015 at 8:23 AM, Emily Stamey <notifications@github.com>
wrote:

> In INSTALL.md
> <https://github.com/mdwheele/vm/pull/54#discussion_r32006705>:
>
> > @@ -8,6 +8,10 @@ INSTALL/UPGRADE VM
> >
> >      > vagrant plugin install vagrant-vbguest
> >
> > +    Update your dotfiles in your own `~` directory
> > +
> > +    Update the `config.dist.yaml` file as needed (primarily memory and shared files location)  There is no longer a merge between config files.
>
> But on Friday that was not the case, was it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mdwheele/vm/pull/54/files#r32006705>.
>



-- 
Dustin Wheeler
Web Application Developer
College of Engineering, ITECS
North Carolina State University
Raleigh, NC 27695, x5-9786

mdwheele@ncsu.edu

Find my office with Google Maps:
http://go.ncsu.edu/find-dustin